lib/trino-parquet/src/main/java/io/trino/parquet/predicate/DictionaryDescriptor.java

```diff
@@ -21,11 +21,13 @@
 public class DictionaryDescriptor
```

Comment:
> When the nulls count in column statistics for a column is 0

Do you think it's possible for some writer to set it incorrectly?

Reply:
If column statistics are incorrect then our existing predicate pushdown logic could also give wrong results, so we should be able to rely on it